iis-assembly-resources/README.markdown

```diff
@@ -1,5 +1,7 @@
-Module utilized by `icm-iis-parent-container`.
+Module utilized by `iis-parent-container`.
```

Comment:
Ok. I'll also search through the code for other occurrences of `iis-parent-container` and change them to `iis-workflows`.